src/OpenIddict.Core/OpenIddictProvider.Authentication.cs

```diff
@@ -180,12 +202,17 @@ public partial class OpenIddictProvider<TUser, TApplication> : OpenIdConnectServ
             ticket.SetResources(context.Request.GetResources());
             ticket.SetScopes(context.Request.GetScopes());
 
+            logger.LogDebug("AuthenticationTicket generated succesfully.");
+
             // Call SignInAsync to create and return a new OpenID Connect response containing the serialized code/tokens.
             await context.HttpContext.Authentication.SignInAsync(ticket.AuthenticationScheme, ticket.Principal, ticket.Properties);
 
+            logger.LogDebug("SignInAsync called.");
```

Comment:
I'm thinking about removing it because we already logged `logger.LogInformation("The authorization request was successfully handled.");`
What about you?